lib/content-services/src/lib/content-node-selector/content-node-selector.component.ts

```diff
@@ -179,16 +198,18 @@ export class ContentNodeSelectorComponent implements OnInit {
     }
 
     getWarningMessage(): string {
-        return this.showingSearch ? 'NODE_SELECTOR.UPLOAD_BUTTON_SEARCH_WARNING_MESSAGE' :
-            (this.hasNoPermissionToUpload() ? 'NODE_SELECTOR.UPLOAD_BUTTON_PERMISSION_WARNING_MESSAGE' : '');
+        return this.showingSearch
```

Comment:
Looks like this is treated as code smell by sonarcloud. Maybe should we improve that code?
<img width="850" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/bd1a86f4-0f6b-46c2-8fb6-10e4189e45c2">